headers/types/ground_mode/ground_mode.h

```diff
@@ -0,0 +1,19 @@
+// Types used during the ground part (overlay 11)
+
+#ifndef HEADERS_TYPES_GROUND_MODE_H_
+#define HEADERS_TYPES_GROUND_MODE_H_
+
+#include "partner_following.h";
+
+// A global structure holding various pointer to important structure
```

Comment:
Can we elaborate a bit more? How about "A global structure holding various pointers to important structures for ground mode"?